src/enums/reviewSummaryFlags.ts

```diff
@@ -0,0 +1,7 @@
+/** Enum for validator flags. */
+export enum ReviewSummaryFlags {
```

Comment:
Please add comment that these are used as selectors and need to match the section IDs.

Reply:
ok

Reply:
Comment updated in next commit.